src/main/webapp/WEB-INF/tags/instructor/course/studentTableModal.tag

```diff
@@ -4,10 +4,10 @@
   <div class="modal-dialog modal-lg">
     <div class="modal-content">
       <div class="modal-header">
-        <span class="pull-left help-block">
-          Tips: After Selecting the table, <kbd>Ctrl + C</kbd> to COPY and <kbd>Ctrl + V</kbd> to PASTE to your Excel Workbook.
+        <span class="help-block">
```

Comment:
What are the effects of removing `pull-left`?

Reply:
pull-left is a bootstrap CSS class that floats the span left, I think this was done to have a better appearance with the "close" and "select table" buttons on the same line as the tip. 

Since the tip is longer it looked awkward with the buttons over on the side so far so I decided to have them on the line below. In that case, I think it's better just to have the tip text over the buttons. So I just thought it looks better. I understand though that it might not fit if you have a specific set of rules for design stuff.

(There are pictures of the original and my change in the issue comments for reference.)

Reply:
Apologies; I only just closely reviewed the UI change by pulling your code and viewing the dialog in question. A suggestion I have is to use a small cross on the top right of the dialog instead of a close button, and a link instead of a select button. This would help to save space and results in a more streamlined appearance:
<img width="908" alt="screen shot 2017-10-27 at 4 39 03 pm" src="https://user-images.githubusercontent.com/6811428/32095616-504b32c0-bb36-11e7-93c8-38283861c722.png">
You can refer to the edit view dialog for the required HTML to implement the top right cross.

While this is a lot more than we typically demand of a first time contributor, would you be willing to give it a shot? You seem to be pretty proficient in this :)

Reply:
That is a good suggestion and I agree that would be an improvement. Yes, I'd be willing to take on that task. Thanks :)